target/linux/ath79/dts/qca9558_sitecom_wlr-8100.dts

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	model = "Sitecom WLR-8100 (X8 AC1750)";
+	compatible = "sitecom,wlr-8100", "qca,qca9558";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &status_amber;
+		led-failsafe = &status_amber;
+		led-running = &status_amber;
+		led-upgrade = &status_amber;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wifi2g {
+			label = "2.4GHz - RFKILL";
+			gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+
+		wifi5g {
+			label = "5GHz - RESTART";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status_amber: status_amber {
```

Comment:
Please include "led_" in the label, so
`led_status_amber: status_amber {`

Reply:
Fixed